packages/sdk/src/Confidence.ts

```diff
@@ -38,6 +38,8 @@ export interface ConfidenceOptions {
   timeout: number;
   /** Debug logger */
   logger?: Logger;
+  /** Backend only option. Sets an alternative resolve url */
+  APIResolveBaseUrl?: string;
```

Comment:
@andreas-karlsson is there some typescript way where we can make it visible in the API that `region` will not be respected if one provides the `customResolveBaseUrl`?

Reply:
Nice! And that wouldn't even be a breaking change to the API, right? Because it unpacks the types?

Reply:
That would be in the API itself?